### ISSUE TITLE
Possibility to reopen the Cookie banner during a session

### ISSUE THREAD

**Issue by mdellming** [hajkmap/Hajk#1125]
Great to have a cookie banner! 👍
But today it is only visible at start, and you can choose between necessary or functional cookies. Sometimes you may want to make a new choice during the session but there is no functionallity for doing so on a general level. This possibility does not exist other than as an embedded function at tool level today (Draw?).

Ideally, we would like to have the possibility to easily get the cookie-banner back if you want to choose a different storage level for the cookies, during the ongoing session. 
If possible, we would like to see this function on a more general level than the one for the draw-tool, such as a tool button with a cookie-symbol.
When you click the tool button the banner will be visible in the bottom of the page again and you will have the possibility to choose a different level, functionality as is today.

Another solution could be to make it possible as part of a "general settings" tool button, if you want to collect these as a tool.

**Comment by jacobwod** [comment]
Good idea!

Should be rather easy to implement given that the functionality is there (as you mentioned, it's used in Sketch). 

The main question is, in my opinion, if this should be a control button (among others, such as the zoom buttons), or if it should be part of a global _Settings_ dialog. I think I opt for the second solution, mainly because we have more and more settings that could be exposed as user settings. Which leads us into a bigger job than just adding this as a simple button that toggles the cookie settings dialog. 

Anyway, good idea and I'm open for discussion.

**Comment by Hallbergs** [comment]
I agree that this should be placed under some global-settings-dialog (not implemented yet). We could move the theme-toggler (and probably other control-buttons) in there to free up some room.

**Comment by mlromramse** [comment]
I would like to vote this issue up and to vote for the cookie-symbol rather than global-settings since the law states "Make it easy for the user to revoke the cookie consent". If it is a plugin it can also easily be removed for inhouse implementations without the need for cookie consent.
A cookie-symbol would also, fairly easily, solve the urgent need to comply with the law and does not in any way crash with the initiative of a global setting in the future.
IMHO the cookie symbol/plugin is what is missing to comply with the law.

To minimize any misunderstandings the cookie-symbol should display the cookie banner when clicked on. Nothing more. The cookie banner itself is easy enough to make site personalized.

**Comment by Hallbergs** [comment]
Valid point @mlromramse. Feel free to implement!
